talisker/wsgi.py

```diff
@@ -45,6 +48,30 @@
 ]
 
 
+class WSGIMetric:
+    latency = talisker.metrics.Histogram(
+        name='wsgi_latency',
+        documentation='Duration of requests served by WSGI',
+        labelnames=['view', 'status', 'method'],
+        statsd='{name}.{view}.{method}.{status}',
+        buckets=[4, 8, 16, 32, 64, 128, 256, 512, 1024, 2048, 4096, 8192],
+    )
+
+    count = talisker.metrics.Counter(
+        name='wsgi_count',
```

Comment:
Minor, but I kind of dislike this name. `wsgi_request_count` maybe?

Reply:
Similar with the parameter name. `count` that is a `Counter` isn't very descriptive of what it's counting.